area_width: 50.0
area_height: 50.0
grid_step: 5.0
flight_altitude: 10.0
